kernel/src/main/java/org/kframework/compile/checks/CheckAtt.java

```diff
@@ -58,6 +59,18 @@ public void check(Sentence sentence) {
         } else if (sentence instanceof Production) {
             check((Production) sentence);
         }
+        checkLabel(sentence);
+    }
+
+    private static final Pattern whitespace = Pattern.compile("\\s");
```

Comment:
Why didn't you include the backtick here?
You have to traverse the label twice now.

Reply:
it's still linear time, the string is unlikely to be long, and I doubt the impact on performance will be substantial. We can always address this later if need be. I suspect the amdahl's law on this will be pretty minimal though.